Library/Formula/llvm.rb

```diff
@@ -116,18 +152,66 @@ def install
     # Apple's libstdc++ is too old to build LLVM
     ENV.libcxx if ENV.compiler == :clang
 
-    if build.with?("lldb") && build.without?("clang")
-      raise "Building LLDB needs Clang support library."
+    if build.with? "clang"
+      (buildpath/"tools/clang").install resource("clang")
     end
 
-    if build.with? "clang"
+    if build.with? "extra-tools"
+      if build.with? "clang"
+        (buildpath/"tools/clang/tools/extra").install resource("extra-tools")
+      else
+        odie "clang's extra tools must be built with clang"
+      end
+    end
+
+    if build.with? "libcxx"
       (buildpath/"projects/libcxx").install resource("libcxx")
-      (buildpath/"tools/clang").install resource("clang")
-      (buildpath/"tools/clang/tools/extra").install resource("clang-tools-extra")
+      (buildpath/"projects/libcxxabi").install resource("libcxxabi")
```

Comment:
I was thinking it could also be useful for cross-compiling to *nix or something, but libc++abi is only supported by OS X at the moment, and the webpage doesn't mention anything about ports to other platforms... Worth leaving in?